projects/go-lib/src/lib/components/go-copy/go-copy.component.ts

```diff
@@ -7,6 +7,9 @@ import { Component, ElementRef, Input, ViewChild } from '@angular/core';
   styleUrls: ['./go-copy.component.scss']
 })
 export class GoCopyComponent {
+
+  @HostBinding('class') class: string = 'go-copy';
```

Comment:
Will this prevent someone from adding their own class to the component? Should we set it like this instead?
```
@HostBinding('class.go-copy')
```

Reply:
Yeah, this is a good catch as well. I switched this around. Thanks